test/e2e/network/service.go

```diff
@@ -100,18 +100,19 @@ var _ = SIGDescribe("Services", func() {
 	// TODO: We get coverage of TCP/UDP and multi-port services through the DNS test. We should have a simpler test for multi-port TCP here.
 
 	/*
-	   Testname: service-kubernetes-exists
-	   Description: Make sure kubernetes service does exist.
+		Release : v1.9
+		Testname: Kubernetes Service
+		Description: By default when a kubernetes cluster is running there SHOULD be a ‘kubernetes’ service running in the cluster.
 	*/
 	framework.ConformanceIt("should provide secure master service ", func() {
 		_, err := cs.CoreV1().Services(metav1.NamespaceDefault).Get("kubernetes", metav1.GetOptions{})
 		Expect(err).NotTo(HaveOccurred())
 	})
 
 	/*
-		    Testname: service-valid-endpoints
-		    Description: Ensure a service with no pod, one pod or two pods has
-			valid/accessible endpoints (same port number for service and pods).
+		Release : v1.9
+		Testname: Service, endpoints
+		Description: Create a service with a endpoint without any Pods, the service MUST run and show empty endpoints. Add a pod to the service and the service MUST validate the ports exposed. Add another Pod then the list of all Ports exposed by both the Pods MUST be validated. Once the second Pod is deleted then set of endpoint SHOULD be validate to show only porsts form the first container are exposed. Once both pods are deleted the endpoints SHOULD be empty.
```

Comment:
Again, the use of SHOULD here is incorrect.

Additionally, I would argue that the original Description is more useful to a developer trying to debug a failing test than the new and "improved" one is...